README.md

```diff
@@ -23,8 +23,8 @@ with dashes (`-`) replaced with underscores (`_`).
 `cluster-domain` | `cluster.local` | Domain name of the cluster. Optional.
 `pod-ip-tags` | `kubernetes,k8s-pod` | Comma-separated list of tags to add to pod IPs in NetBox. Any tags that don't yet exist will be created. Optional.
 `service-ip-tags` | `kubernetes,k8s-service` | Comma-separated list of tags to add to service IPs in NetBox. Any tags that don't yet exist will be created. Optional.
-`pod-publish-labels` | `app` | Comma-separated list of kubernetes pod labels to be added to the IP description in NetBox in `label: label_value` format. Optional. 
-`service-publish-labels` | `app` | Comma-separated list of kubernetes service labels to be added to the IP description in NetBox in `label: label_value` format. Optional. 
+`pod-publish-labels` | `app` | Comma-separated list of kubernetes pod labels to be added to the IP description in NetBox in `label: label_value` format. Only the IPs of the pods that have one of these labels set will be exported. Optional. 
+`service-publish-labels` | `app` | Comma-separated list of kubernetes service labels to be added to the IP description in NetBox in `label: label_value` format. Only the IPs of the services that have one of these labels set will be exported. Optional. 
```

Comment:
```suggestion
`pod-publish-labels` | `app` | Comma-separated list of kubernetes pod labels to be added to the IP description in NetBox in `label: label_value` format. Only the IPs of the pods that have at least one of these labels set will be exported. Optional. 
`service-publish-labels` | `app` | Comma-separated list of kubernetes service labels to be added to the IP description in NetBox in `label: label_value` format. Only the IPs of the services that have at least one of these labels set will be exported. Optional. 
```

Reply:
You're right :thinking: that could be quite a frustrating detail to miss and than have to figure out